### PR TITLE
Task 3870  evening report loadmore button

### PR DESCRIPTION
**Pull request by TerryKolodii**
https://redmine.devbranch.work/issues/3870

### CODE REVIEW

File: web/themes/custom/news/src/styles/entry.scss [hunk 1]
```diff
@@ -446,17 +448,22 @@
       .views-row {
         position: relative;
         height: 50%;
+
         .views-field-field-news-category {
           position: absolute;
           top: 0%;
           left: 5%;
           background: #3d348b;
           padding: 0 3%;
```
Comment (henzzapp) [inline web/themes/custom/news/src/styles/entry.scss]: NOTE: Bad practice to use % in spacings

File: web/themes/custom/news/src/styles/entry.scss [hunk 2]
```diff
@@ -470,30 +477,56 @@
...
+
+      .row-5 {
+        display: none;
+      }
     }
   }
-  .more-link {
+
+  .pager {
+    margin-top: 10px;
```
Comment (henzzapp) [inline web/themes/custom/news/src/styles/entry.scss]: @TerryKolodii do you have a spacing() function in SCSS already?

File: web/themes/custom/news/src/styles/entry.scss (hunk 1 above)
Comment (TerryKolodii) [reply web/themes/custom/news/src/styles/entry.scss]: fixed

File: web/themes/custom/news/src/styles/entry.scss (hunk 2 above)
Comment (TerryKolodii) [reply web/themes/custom/news/src/styles/entry.scss]: Now i have. Edited styles using it

File: web/themes/custom/news/src/styles/entry.scss [hunk 3]
```diff
@@ -260,181 +260,187 @@
...
+//          width: 100%;
+//        }
+//      }
+//    }
+//    .paragraph--type--editor-picks-large {
+//      position: relative;
+//      .field--name-field-news-category {
+//        position: absolute;
+//        top: 30%;
+//        left: 18px;
```
Comment (andriy-tkachuk) [inline web/themes/custom/news/src/styles/entry.scss:342]: Remove this if not needed

**Review by andriy-tkachuk — CHANGES_REQUESTED** [review]

**Review by henzzapp — APPROVED** [review]

File: web/themes/custom/news/src/styles/entry.scss (hunk 3 above)
Comment (TerryKolodii) [reply web/themes/custom/news/src/styles/entry.scss:342]: > Remove this if not needed

Done

Comment (andriy-tkachuk) [reply web/themes/custom/news/src/styles/entry.scss:342]: можеш не коментувати тут, я і так побачу що ти видалив. Або просто можеш тицьнути кнопку Resolve conversation нижче

**Pull request merged**